### ISSUE TITLE
What license does this library have?

### ISSUE THREAD

**Issue by VictorTrapenok** [NuclearC/ini-file#1]
Hello, what license does this library have?
Can i use this library in commerce project?

**Comment by NuclearC** [comment]
Yes you can use but now it uses some Windows API's so its not really cross platform.
For license i would say its GPL.
